python/py-libxml2/Portfile

```diff
@@ -18,16 +18,17 @@ long_description    ${description}
 
 homepage            http://xmlsoft.org/python.html
 
-master_sites        http://www.xmlsoft.org/sources/ \
-                    ftp://gd.tuwien.ac.at/pub/libxml/ \
-                    ftp://xmlsoft.org/libxml2/
+master_sites        https://download.gnome.org/sources/libxml2/2.9/
```

Comment:
is this not the new, official download site for the project?

Reply:
Er, yes?

Reply:
You're right it isn't. I struggled until I found the e-mail.